docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+
+For Gammapy, by default, the maintainers are the Lead Developers. If in the future a task is dedicated to the creation
+of a release, the maintainer will be the person in charge of this task.
+
+Releases
+--------
+
+Each Gammapy release should be associated with an updated list of authors. The list of authors is composed by
+people who contributed as listed in the previous sub-section to the current, or in the previous release in
+order to account for long contributions (e.g. management of depreciated features, improvement/debugging of features,
+etc). Any additional author can be inserted into the list for its special contribution when requested by the Gammapy
+Coordination Committee.
+
+The order of the authors is first 'The Gammapy Collaboration', the Lead Developers, a past Lead Developer in case of
+change between two releases, followed by the list of contributors in alphabetical order.
+
+This policy is applied to any release after the LTS V1.0. For the releases made before, a hand-made list is used.
+After the mention of 'The Gammapy Collaboration', the current Lead Developers will be in first, then the past Lead
+Developers, followed by the list of contributors in alphabetical order.
+
+Long Term Support releases
+--------------------------
+
+The list of authors is composed of the union of all the contributors of the releases realised up to the last LST
+release. Any additional author can be inserted into the list for its special contribution when requested by the Gammapy
+Coordination Committee.
+
+The order of the authors is first 'The Gammapy Collaboration', the Lead Developers, a past Lead Developer in case of
+change since the last LST, followed by the list of contributors in alphabetical order.
+
+For the first LST release, the V1.0, all contributors from the beginning of the project will be co-authors. After the
+mention of 'The Gammapy Collaboration', the Lead Developers will be in first, then the past Lead Developers since the
+beginning of the project, followed by the list of contributors in alphabetical order.
+
+General Gammapy publications
+----------------------------
+
+This product aims to describe the project and/or the library as a whole and is submitted to peer-review journals. As it
+targets a wide community, the following scheme is used.
+
+A/ List of authors
+
+A.1/ By default, 'The Gammapy Collaboration' is mentioned first, then the Lead Developers, then the past Lead Developers since the
+last general Gammapy publication, then the list of contributors of each release since the last Gammapy publication.
+
+A.2/ If the editorial rules of the targeted journal permit it, the scheme used by the Astropy project
+(e.g. `Astropy V2.0 <https://arxiv.org/pdf/1801.02634.pdf>`_) should be used in priority:
+
+- 'The Gammapy Collaboration', and the list of primary paper contributors in order of contribution agreed per consensus with the development team and the Coordination Committee, as '(Primary Paper Contributors)'
```

Comment:
I'd rather go for gammapy team rather than collaboration.

Reply:
+1 For "Gammapy Team" or "Gammapy Developers"

Reply:
ok